### PR TITLE
[9.x] Document multiple subscriptions in Cashier

### PR DESCRIPTION
**Pull request by driesvints**
For both Paddle and Stripe.

### CODE REVIEW

File: billing.md [hunk 1]
```diff
@@ -1106,6 +1107,52 @@ You can also retrieve a specific price using the `findItemOrFail` method:
 
     $subscriptionItem = $user->subscription('default')->findItemOrFail('price_chat');
 
+<a name="multiple-subscriptions"></a>
+### Multiple Subscriptions
+
+Stripe allows your customers also to have multiple subscriptions. For example, you might run a sports service that offers a swimming subscription and a gym subscription. Depending on how frequently you come you can have different prices for each of these types of subscriptions. Of course, customers should be able to subscribe to either one or both. In you application you can make the differentiation between these two types of subscriptions by making use of the `$name` parameter when referencing subscriptions.
```
Comment (jayanratna) [inline billing.md:1113]:
```suggestion
Stripe allows your customers to have multiple subscriptions. For example, you might run a sports service that offers a swimming subscription and a gym subscription. Depending on how frequently you come, you can have different prices for each of these types of subscriptions. Of course, customers should be able to subscribe to either one or both. In your application you can make the differentiation between these two types of subscriptions by making use of the `$name` parameter when referencing subscriptions.
```

File: billing.md [hunk 2]
```diff
@@ -1106,6 +1107,52 @@ You can also retrieve a specific price using the `findItemOrFail` method:
...
+        $request->user()->newSubscription('swimming')
+            ->price('price_swimming_monthly')
+            ->create($request->paymentMethodId);
+
+        // ...
+    });
+
+Notice how the first parameter of `newSubscription` is now `swimming`. This is to differentiate from other types of subscriptions. Usually in most services there's only one type of subscription and then this can stay to its `default` value.
+
+Using a different type of subscription will allow you to handle plan changes separately without affecting the other subscription. Thing such as invoicing and billing cycle will also differ for these two type of subscriptions.
```
Comment (jayanratna) [inline billing.md:1129]:
```suggestion
Using a different type of subscription will allow you to handle plan changes separately without affecting the other subscription. Tasks such as invoicing and billing cycles will also differ for these two type of subscriptions.
```

I feel that "tasks" is more appropriate here compared to "thing"

File: cashier-paddle.md [hunk 3]
```diff
@@ -855,6 +856,52 @@ If you wish to cancel a subscription immediately, you may call the `cancelNow` m
 > **Warning**  
 > Paddle's subscriptions cannot be resumed after cancellation. If your customer wishes to resume their subscription, they will have to subscribe to a new subscription.
 
+<a name="multiple-subscriptions"></a>
+### Multiple Subscriptions
+
+Paddle allows your customers also to have multiple subscriptions. For example, you might run a sports service that offers a swimming subscription and a gym subscription. Depending on how frequently you come you can have different prices for each of these types of subscriptions. Of course, customers should be able to subscribe to either one or both. In you application you can make the differentiation between these two types of subscriptions by making use of the `$name` parameter when referencing subscriptions.
```
Comment (jayanratna) [inline cashier-paddle.md:862]:
```suggestion
Paddle allows your customers to have multiple subscriptions. For example, you might run a sports service that offers a swimming subscription and a gym subscription. Depending on how frequently you come, you can have different prices for each of these types of subscriptions. Of course, customers should be able to subscribe to either one or both. In your application you can make the differentiation between these two types of subscriptions by making use of the `$name` parameter when referencing subscriptions.
```

File: cashier-paddle.md [hunk 4]
```diff
@@ -855,6 +856,52 @@ If you wish to cancel a subscription immediately, you may call the `cancelNow` m
...
+        $request->user()->newSubscription('swimming')
+            ->price($swimmingMonthly = 12345)
+            ->create($request->paymentMethodId);
+
+        // ...
+    });
+
+Notice how the first parameter of `newSubscription` is now `swimming`. This is to differentiate from other types of subscriptions. Usually in most services there's only one type of subscription and then this can stay to its `default` value.
+
+Using a different type of subscription will allow you to handle plan changes separately without affecting the other subscription. Thing such as invoicing and billing cycle will also differ for these two type of subscriptions.
```
Comment (jayanratna) [inline cashier-paddle.md:878]:
```suggestion
Using a different type of subscription will allow you to handle plan changes separately without affecting the other subscription. Tasks such as invoicing and billing cycles will also differ for these two type of subscriptions.
```

**Review by jayanratna — COMMENTED** [review]
This addition will be very useful for anyone who didn't know Cashier could do this 😉

Made a few suggestions. 🙂

**Comment by driesvints** [comment]
Thanks @jayanratna

**Pull request merged**